include/librealsense2/hpp/rs_processing.hpp

```diff
@@ -224,19 +224,39 @@ namespace rs2
         frame_queue _queue;
     };
 
-    class syncer
+    class syncer_processing_block
     {
     public:
-        syncer()
+        syncer_processing_block()
```

Comment:
I prefer asynchronous_syncer, or syncer_with_callbacks over this. It's not descriptive enough to be an API.